docs/dataset/object-detection.md

```diff
@@ -0,0 +1,366 @@
+        unit square. The greater the area under the curve, the more performant your model is. Typically, there is
+        a tradeoff between precision and recall, which you can read in detail [here](../metrics/pr-curve.md).
+
+        <center>
+        ![pr.png](../assets/images/metrics-prcurve-light.png#only-light)
+        ![pr.png](../assets/images/metrics-prcurve-dark.png#only-dark)
+        </center>
+
+    2. [**Confusion Matrix**](../metrics/confusion-matrix.md): Particularly for multiclass object detection, a
+        confusion matrix displays the actual classes against the predicted classes in a table. This allows for a
+        clear understanding of the model's performance with respect to classifying classes.
+
+        <center>
+        ![pr.png](../assets/images/metrics-confusion-matrix-light.png#only-light)
+        ![pr.png](../assets/images/metrics-confusion-matrix-dark.png#only-dark)
+        </center>
+
+        From the plot above, we can see if the model is confused when detecting cats and dogs.
+
+    Note that any plot suitable for a classification task is also suitable for an object detection task, as object
+    detection is a combination of a classification and localization task. However, it is important to consider what
+    matters to your project and design custom plots to describe your model's performance in the best way possible.
+    For example: an F1-Score vs confidence threshold plot, a histogram of IoUs, or mAP vs different IoUs.
+
+Plots can become very powerful ways to gain insights into unexpected model behavior, and a formal way to showcase
+strong model quality.
+
+??? info "Advanced Plot Insights"
+
+    For `person object detection`, it would be interesting to see
+    how performance differs by different test cases or different characteristics of data. With different lines on
+    plots representing different groups, such as `body part`, `race`, `bounding box size`, `image brightness`, etc.
+    plots easily explain where performance drops.
+
+    <center>
+    ![pr_by_testcase.png](../assets/images/task-od-prcurve-light.png#only-light)
+    ![pr_by_testcase.png](../assets/images/task-od-prcurve-dark.png#only-dark)
+    </center>
+
+    From the example above, we see how a particular model suffers in performance when only a
+    person's `arm` is shown.
+
+## Using Kolena to Evaluate Object Detection Models
+
+In this guide, we will use the [COCO 2014](https://cocodataset.org/#home) dataset to demonstrate test data curation
+and model evaluation in Kolena for object detection, using both the web app and the `kolena` Python SDK.
+In the examples below, using the web app will walk you through a single class object detection task,
+and following the SDK instructions will demonstrate a multiclass object detection task, just for some variety.
+
+Kolena makes object detection model evaluation very easy in just three simple steps:
+
+### Step 1: Upload an Object Detection Dataset
```

Comment:
I think step 1 should be formatting their dataset into kolena compatible dataframe / csv / jsonl / parquet file.  Should we have the `Generating Datasets` callout section as the step 1?